helm-charts/common/lvm-serving/README.md

```diff
@@ -0,0 +1,56 @@
+# LVM Serving Microservice (lvm-serving)
+
+Helm chart for deploying LVM-Serving microservice.
+
+`lvm-serving` is a microservice which provides inference from Video-Llama-2-7b.
+
+## Installing the Chart
+
+To install the chart, run the following:
+
+```bash
+cd GenAIInfra/helm-charts/common
+export HFTOKEN="insert-your-huggingface-token-here"
+
+# Set a dir to cache downloaded Video-Llama Model
+export MODELDIR=/mnt/opea-models
```

Comment:
Using persistent volumes better than a node level directory. We have some charts with PV support. I believe submitted by @poussa

Reply:
any possibility, we can take this up in future in further improvements to the videoqna chart?